app/move/views/_includes/person-escort-record-summary.njk

```diff
@@ -5,26 +5,11 @@
     </h2>
 
     {% if assessment.isCompleted %}
-      {% for panel in assessment.panels %}
-        {% call appPanel(panel) %}
-          {{ appMetaList(panel.metaList) }}
-        {% endcall %}
-      {% else %}
-        {{ appMessage({
-          classes: "app-message--muted govuk-!-margin-top-2",
-          allowDismiss: false,
-          content: {
-            html: t("person-escort-record::flags.empty", {
-              name: assessment.name,
-              url: assessment.url
-            })
-          }
-        }) }}
-      {% endfor %}
+      {{ renderAssessmentComponent(assessment) }}
```

Comment:
The main benefit of this change is that it will allow a youth risk assessment section to be set as a `previousAssessment` and to be rendered correctly.